include/blmc_robots/blmc_joint_module.hpp

```diff
@@ -401,10 +408,22 @@ class BlmcJointModules
         }
     }
 
+    /**
+     * @brief Set the polarities of the joints
+     * 
+     * @param polarities 
+     */
+    void reverse_joint_polarities(std::array<bool, COUNT> polarities)
+    {
+        for(size_t i = 0; i < COUNT; i++)
+        {
+            modules_[i]->reverse_joint_polarity(polarities[i]);
+        }
+    }
     /**
      * @brief Register the joint torques to be sent for all modules.
      * 
-     * @param desired_torques 
+     * @param desired_torques
```

Comment:
Please provide unites.

Reply:
I will do this, though this is not really the objective of this (I guess I do not like trailing space)